src/models/SentimentModel.ts

```diff
@@ -4,4 +4,8 @@ import RekognitionModel from './RekognitionModel';
 export default class SentimentModel extends BaseModel {
   constructor (private readonly rekognitionModel: RekognitionModel) {
     super();
-  }}
+  }
+  analyzeImages (images : string[]) {
+    return this.rekognitionModel.detectFaces(images);
```

Comment:
here you probably want to eventually add the magic rather than directly returning the output from detect faces, right?